l1/l1.go

```diff
@@ -56,7 +60,30 @@ func (c *Client) subscribeToUpdates(ctx context.Context,
 	return sub, nil
 }
 
+func (c *Client) checkChainID(ctx context.Context) error {
+	gotChainID, err := c.l1.ChainID(ctx)
+	if err != nil {
+		return err
+	}
+	if err != nil {
+		return fmt.Errorf("retrieve Ethereum chain ID: %w", err)
+	}
```

Comment:
what is going on here?

Reply:
Forgot to remove the first error. Fixed, thanks.